sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -2638,6 +2641,705 @@ SUBROUTINE CALGRADPS(PS,PSX,PSY)
 !     END IF 
 
       END SUBROUTINE CALGRADPS
+
+!> calslr_roebber() computes snow solid-liquid-ratio slr using the Roebber algorithm
+!>
+!> ### Program history log:
+!> Date | Programmer | Comments
+!> -----|------------|---------
+!> 2022-07-11 | Jesse Meng | Initial
+!>
```

Comment:
@JesseMeng-NOAA suggest document here what you described this morning about how WPC generates 3 versions of SLR, and also outputs the average but EMC will only outputs model forecasted ones.

Reply:
Updated